test-case-reporting/test/test_result_record.test.ts

```diff
@@ -316,4 +316,12 @@ describe('TestResultRecord', () => {
       expect(testRuns).toMatchObject(sampleTestRuns);
     });
   });
+
+  describe('getTestRunsOfBuild', () => {
+    it.todo('gets only the test runs of the build');
+  });
+
+  describe('getTestCaseHistory', () => {
+    it.todo('gets the test case history in the right order');
```

Comment:
What is "the right order"? :)

Reply:
Good point -- it's reverse chronological order.

Done improving test description.